Donell/exceptions_challenge.py

```diff
@@ -0,0 +1,71 @@
+
+# Question 1: validate_user_input
+
+from xml.dom import ValidationErr
+
+
+def validate_user_input():
+    user_number = validate_user_input
+    user_number(input("please enter a number"))
+        try:
+            int(user_number)
+            print("your input is valid")
```

Comment:
If you print "your input is valid" here, this line will always be printed even if the user puts the correct input. Please go back to the lecture and review the try except session.